content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -203,9 +203,11 @@ when both the following statements are true:
 To facilitate this change to Pod readiness evaluation, a new Pod condition
 `ContainersReady` is introduced to capture the old Pod `Ready` condition.
 
-As an alpha feature, the "Pod Ready++" feature has to be explicitly enabled by
+In K8s 1.11, as an alpha feature, the "Pod Ready++" feature has to be explicitly enabled by
 setting the `PodReadinessGates` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
```

Comment:
@freehan seems like the referenced doc also needs to update accordingly, right?

Reply:
👍

Reply:
Thanks! Fixed!

Reply:
LGTM :+1: